.github/labeler.yml

```diff
@@ -1,104 +1,69 @@
+"area: crates":
+  - "crates/**/*"
+
 "area: dependencies":
-  - "**/patches/*"
-  - "**/patches/.*"
   - "**/patches/**/*"
-  - "**/patches/**/.*"
   - "**/yarn.lock"
 
 "area: hub":
-  - "hub/*"
-  - "hub/.*"
   - "hub/**/*"
```

Comment:
Please liaise with @CiaranMn to remove the `hub` folder in its entirety now we support `npm`-based publishing. 

Side-note: designs for a Git-based publication flow are also complete (but the corresponding dev work not yet planned). In any case, we don't want to be maintaining inbound PRs from users containing their blocks if we can avoid it. The `hub` folder and its contents should be removed once their contents have been mirrored into our block-serving solution.